crates/solver/src/driver.rs

```diff
@@ -468,7 +471,12 @@ impl Driver {
             };
 
             for settlement in &settlements {
-                tracing::debug!("solver {} found solution:\n{:?}", name, settlement);
+                tracing::debug!(
+                    "solver {} found solution:\n{:?} for auction id: {:}",
+                    name,
+                    settlement,
+                    auction_id
```

Comment:
ubernit: I would reorder the line so that the the ID is near the front. The `{:?}` debug print of the settlement data is very long so it makes it hard to see the auction ID when scanning the logs.

```suggestion
                    "solver {name} found solution for auction {auction_id}:\n{settlement:?}",
```